### PR TITLE
Fix ZZZ & Add custom header functionality

### LINKED ISSUE #52: Zenless Zone Zero Login returns error
After attempting to login or run the script , ZZZ login returns the following message in discord via webhook and login does not reflect in the login webpage. All other games' login is working as intended.
 
 - 活动已结束 (-500012)

### PR DESCRIPTION
**Pull request by WiLuX-Source**
Fixes #52

### CODE REVIEW

**Comment by WiLuX-Source** [comment]
Tested discord english version.
Other versions needs testing but should be fine.

File: src/main-telegram_zh-tw.gs [hunk 1]
```diff
@@ -43,46 +73,27 @@ function autoSignFunction({
...
     method: 'POST',
-    headers: header,
     muteHttpExceptions: true,
   };
 
   let response = `${accountName} 的自動簽到作業已完成`;
 
   var sleepTime = 0
   const httpResponses = []
-  for (const url of urls) {
```
Comment (Chillrend) [inline src/main-telegram_zh-tw.gs:76]: You removed this part at your PR, pretty sure it's not intentional.

File: src/main-telegram_zh-tw.gs [hunk 2]
```diff
@@ -43,46 +73,27 @@ function autoSignFunction({
...
-    Utilities.sleep(sleepTime);
-    httpResponses.push(UrlFetchApp.fetch(url, options));
-    sleepTime = 1000;
-  }
```
Comment (Chillrend) [inline src/main-telegram_zh-tw.gs:80]: You removed this from your PR, pretty sure its not intentional..

**Review by Chillrend — CHANGES_REQUESTED** [review]

File: src/main-telegram_zh-tw.gs (hunk 2 above)
Comment (WiLuX-Source) [reply src/main-telegram_zh-tw.gs:80]: Ah that needs fixing thank you.

**Comment by WiLuX-Source** [comment]
Ok should be fine for now. thank you @Chillrend

**Review by Chillrend — APPROVED** [review]
Okay looks good!

**Review by StarPlayingGames — APPROVED** [review]

**Review by rounak-codes — APPROVED** [review]

**Review by itsPG — APPROVED** [review]

**Review by heymeowcat — APPROVED** [review]

**Review by heymeowcat — APPROVED** [review]

**Comment by WiLuX-Source** [comment]
It seems this project is dead and I need to go seperate go direction. If you liked how I coded this PR you might be interested in my version thanks everyone who approved.

**Review by canaria3406 — COMMENTED** [review]
thanks

**Pull request merged**